bootstrap/bootstrap-storage.sh

```diff
@@ -40,34 +41,60 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-SCRIPT_RUNNER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.SQLScriptRunner
+STREAMLINE_TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.StreamlineTableInitializer
 for file in "${BOOTSTRAP_DIR}"/lib/*.jar;
 do
     CLASSPATH="$CLASSPATH":"$file"
 done
 
 echo "Using Configuration file: ${CONFIG_FILE_PATH}"
 
-function streamlineBootstrapStorage {
-    SCRIPT_DIR="${BOOTSTRAP_DIR}/sql/<dbtype>"
-    FILE_OPT="-f ${SCRIPT_DIR}/drop_tables.sql -f ${SCRIPT_DIR}/create_tables.sql"
+function dropStreamlineTables {
+    ${JAVA} -Dstreamline.bootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${STREAMLINE_TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} --drop-script ${SCRIPT_DIR}/drop_tables.sql --drop
+}
+
+function createStreamlineTables {
+    ${JAVA} -Dstreamline.bootstrap.dir=$BOOTSTRAP_DIR  -cp ${CLASSPATH} ${STREAMLINE_TABLE_INITIALIZER_MAIN_CLASS} -m ${MYSQL_JAR_URL_PATH} -c ${CONFIG_FILE_PATH} --create-script ${SCRIPT_DIR}/create_tables.sql --create
```

Comment:
Also we are expecting sql files to be under bootstrap/sql. Instead of passing all these options I would recommend we follow a syntax.
i.e store all the sql files in this manner
bootstrap/sql/db_type/create_tables.sql , bootstrap/sql/db_type/drop_tables.sql.
we can change the db_type mysql or postegres, phoenix etc..
Instead of having to pass drop_tables.sql or create_tables.sql . You only pass the parent directory under which sql scripts are stored. In this case, it would be "bootstrap/sql".
Your Java Class then can read the streamline  config to determine the right db.type and use that to invoke right db_type sql script and also it can take an argument such as "--create" or "--drop" like you are doing already but no need to pass the specific script to do so.